modules/virt-analyzing-datavolume-conditions-and-events.adoc

```diff
@@ -0,0 +1,87 @@
+it came to be in its current state.
+
+There are many different combinations of `Reason`, `Status`, `Type`, and `Message`
+conditions. Each must be evaluated in its unique context.
+
+Examples of various condition combinations follow.
+
+* `Bound` – If `Type` is `Bound` and `Status` is `True`, then the PVC is `Bound`.
+If the PVC is not bound, the `Status` is `False`. The conditions `Reason` and `Message`
+present details of the current `Status`. In this example, the PVC is `Bound`,
+so `Reason` is `Bound` and `Status` is `True`. The `Message` indicates which
+PVC owns the DataVolume.
++
+`Message`, in the `Events` section, provides further details including how
+long the PVC has bean bound (`Age`) and by what resource (`From`),
+in this case `datavolume-controller`.
++
+----
+Status:
+	Conditions:
+		Last Heart Beat Time:  2020-07-15T03:58:24Z
+		Last Transition Time:  2020-07-15T03:58:24Z
+		Message:               PVC win10-rootdisk Bound
+		Reason:                Bound
+		Status:                True
+		Type:                  Bound
+
+	Events:
+		Type     Reason     Age    From                   Message
+		----     ------     ----   ----                   -------
+		Normal   Bound      24s    datavolume-controller  PVC example-dv Bound
+----
+
+* `Running` – If `Type` is `Running` and `Status` is `True`, an operation is in the
+process of running, such as the ongoing import, in this example.
++
+`Running` will be `False` if the container is not running. If `Ready` is `True`,
+then `Running` will always be `False`, because a DataVolume cannot be ready
+if an import is running.
++
+In the `Events` section, the `Reason` and `Message` contain detailed information
+about why the operation is not running, in addition to the container exit
+message. In this example, the connection failed due to a `404`, listed in the
+`Events` section’s first `Warning`.
++
+----
+Status:
+	 Conditions:
+		 Last Heart Beat Time:  2020-07-15T04:31:39Z
+		 Last Transition Time:  2020-07-15T04:31:39Z
+		 Message:               Import Complete
+		 Reason:                Completed
+		 Status:                False
+		 Type:                  Running
+
+	Events:
+		Type     Reason    Age                From                   Message
+		----     ------    ----               ----                   -------
+		Warning  Error     12s (x2 over 14s)  datavolume-controller  Unable to connect
+		to http data source: expected status code 200, got 404. Status: 404 Not Found
+----
+
+* `Ready` – If `Type` is `Ready` and `Status` is `True`, then the DataVolume is ready
+to be used, as in the following example. If the DataVolume is not ready to be
+used, the `Status` is `False`. If the `Type` is `Ready`, `Reason` and `Message`
+are not displayed.
++
+----
+Status:
+	 Conditions:
+		 Last Heart Beat Time: 2020-07-15T04:31:39Z
+		 Last Transition Time:  2020-07-15T04:31:39Z
+		 Status:                True
+		 Type:                  Ready
+----
```

Comment:
What about using two complete precess (one is import succeed and on is failure) to illustrate the 3 types of conditions and events?  In this way, we can also see the transition to other states.

Reply:
@awels can you comment?

Reply:
Unless it's absolutely needed, I would rather not add more content as we are past freeze on doc, but I am interested to hear Alexander's opinion.

Reply:
Its a trouble shooting guide, not an exhaustive list of variants of which there can be many. I would keep it to a few simple examples.

Reply:
OK, thanks @awels and @qwang1. I am going to do some cleaning on the markup as I think it's a bit cluttered with literals right now (too much markup can have the same affect on the reader as not enough mark-up :-)). 

Will have another version ready to go to peer review today. Will give you both a heads-up, but for now tech content is frozen.

Bob